<launch>
	<param name="use_sim_time" value="true" />

	<!-- Correction of steer bias -->
	<param name="steering_bias" value="7.5" />

	<!-- Monza circuit starting line coordinates -->
	<param name="lat_ref" value="45.618932386592405" />
    <param name="lon_ref" value="9.281178887031235" />
    <param name="alt_ref" value="229" />

	<!-- Sector 1 coordinates -->
	<param name="lat_sec_1" value="45.615024" />
	<param name="lon_sec_1" value="9.280702" />

	<!-- Sector 2 coordinates -->
	<param name="lat_sec_2" value="45.630100" />
	<param name="lon_sec_2" value="9.289733" />

	<!-- Sector 3 coordinates -->
	<param name="lat_sec_3" value="45.623366" />
	<param name="lon_sec_3" value="9.286964" />

	<!-- Nodes execution -->
	<node pkg="first_project" type="odometer" name="odometer_node" output="screen"/>
	<node pkg="first_project" type="gps_to_odom" name="gps_to_odom_server"/>
	<node pkg="first_project" type="gps_odometer" name="gps_odometer_node"/>
	<node pkg="first_project" type="sector_times" name="sector_times_node"/>

	<!-- rviz configuration file -->
  	<node name="rviz" pkg="rviz" type="rviz" args="-d $(find first_project)/rviz/cfg.rviz" />
</launch>